schemas/v3.0/schema.json

```diff
@@ -0,0 +1,1254 @@
+{
+  "title": "A JSON Schema for OpenAPI 3.0.",
+  "id": "http://openapis.org/v3/schema.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "description": "This is the root document object of the OpenAPI definition file.",
+  "required": [
+    "openapi",
+    "info",
+    "paths"
+  ],
+  "additionalProperties": false,
+  "patternProperties": {
+    "^x-": {
+      "$ref": "#/definitions/specificationExtension"
+    }
+  },
+  "properties": {
+    "openapi": {
+      "type": "string"
+    },
+    "info": {
+      "$ref": "#/definitions/info"
+    },
+    "servers": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/server"
+      },
+      "uniqueItems": true
```

Comment:
The uniqueItems constraint brings up an interesting set of issues.  This constraint isn't in the spec. What is the reason for adding it in the Schema?  It seems like it is common sense that the URLs should be unique.  Should that wording be added to the spec?  Can JSON schema validate that the just URLs are unique?